### PR TITLE
fix(Dropdown): change active item on keyboard up/down

### LINKED ISSUE #1487: Dropdown: change active item on keyboard up/down
Hello, do you plan on changing dropdown selection with keyboard keys ?
Just as mentionned in Semantic UI 2.2 news :

**Faster Dropdown Selection**
_Dropdowns now change selections on keyboards without hitting "enter", this lets users quickly navigate between dropdown elements in a form without having to hit "enter" before "tab" after each field entry, saving an additional keystroke._

Thank you !

### PR DESCRIPTION
**Pull request by Bebersohl**
Fixes #1487.

### CODE REVIEW

File: src/modules/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -514,20 +514,19 @@ export default class Dropdown extends Component {
...
+    const moves = {
+      [keyboardKey.ArrowDown]: 1,
+      [keyboardKey.ArrowUp]: -1,
     }
+    const { [keyboardKey.getCode(e)]: move } = moves
+
+    if (move === undefined) return
+    e.preventDefault()
+    this.moveSelectionBy(move)
+    if (!multiple) this.makeSelectedItemActive(e)
```
Comment (layershifter) [inline src/modules/Dropdown/Dropdown.js:529]: I refactored this method to avoid usage of `switch`.

**Review by layershifter — APPROVED** [review]
@Bebersohl Thanks for PR. I pushed a simple refactor there, @levithomason LGTM.

File: src/modules/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -514,20 +514,19 @@ export default class Dropdown extends Component {
...
-      default:
-        break
+    debug('moveSelectionOnKeyDown()', keyboardKey.getName(e))
+
+    const { multiple } = this.props
+    const moves = {
+      [keyboardKey.ArrowDown]: 1,
+      [keyboardKey.ArrowUp]: -1,
     }
+    const { [keyboardKey.getCode(e)]: move } = moves
```
Comment (levithomason) [inline src/modules/Dropdown/Dropdown.js]: This might be a little more clear as:

```jsx
const move = moves[keyboardKey.getCode(e)]
```

**Comment by levithomason** [comment]
Looks great, will merge on pass.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.69.0`

**Pull request merged**